services/tenant-ui/frontend/src/components/messages/Messages.vue

```diff
@@ -21,7 +21,10 @@
         <div class="flex justify-content-between">
           <div class="flex justify-content-start">
             <CreateMessage
-              v-if="config.frontend.showWritableComponents"
+              v-if="
+                config.frontend.showWritableComponents === true ||
+                config.frontend.showWritableComponents === 'true'
+              "
```

Comment:
I think this is probably necessary if the Helm charts are injecting a string from the bool in the yaml.
Ideally the helm chart to Node-Config would result in a boolean but I don't know that off the top of my head...

Maybe ideally some util function for this like
```
function isConfigTrue(value: any): boolean {
  return value === true || value === "true";
}
```

so the logic is encapsulated?

see `services\tenant-ui\frontend\src\helpers\index.ts`

Reply:
That looks pretty slick.

Reply:
I just did a quick search and replace. May use the helper in the future. Thanks for the suggestion @loneil